### ISSUE TITLE
Tab transition callbacks not firing

### ISSUE THREAD

**Issue by adamtay82** [react-bootstrap/react-bootstrap#1907]
For some reason I can never get the callbacks to fire - not sure if I'm doing something stupid?

Code is below - would welcome thoughts!
```
        <Tabs activeKey={this.state.tabKey}
                            id="uncontrolled-tab-example"
                            onSelect={this.handleTabChange}>
             <Tab eventKey={1} title="Snapshot" id="title" onEntered={this.fixupComp} animation={true}>
                            <MyComponent />
              </Tab>
              <Tab eventKey={2} title="Trends" id="trends" onEntered={this.fixupComp} animation={true}>
                           <MyComponent />
              </Tab>
       </Tabs>
```

**Comment by salimhamed** [comment]
I've having the exact same issue.  I'm unable to get any of the callbacks to fire.

**Comment by agilgur5** [comment]
Can confirm this fails for me as well; nothing gets logged what-so-ever:
```
<Tabs style={{width: '624px', float: 'left'}}>
        <Tab eventKey={1} title='Photos' className='top-tabs'>
          ...
        </Tab>
        <Tab eventKey={2} title='Map' className='top-tabs'
          onEntered={() => { console.log('entered'); this.handleResize()}}
          onEnter={() => console.log('enter')}>
          ....
        </Tab>
</Tabs>
```

In my case, I need to fire a resize event (to `react-google-maps`), and this only works properly once the animation has completed...

**Comment by agilgur5** [comment]
Seems to have been fixed in the latest version actually (`0.29.4`), I had been on `0.28.x`
